src/lingpy/convert/html.py

```diff
@@ -425,10 +425,13 @@ def msa2html(
         for i in msa['local']:
             local[i] = '*'
 
+    tokens = msa['seqs']
+    msa['seqs'] = [' '.join(seq) for seq in msa['seqs']]
```

Comment:
Here `msa['seqs']` is reverted back to what it looked like before the PR as code further down relies on it being space-separated strings.